### ISSUE TITLE
[PIMCORE5] Symfony Exception Page executes an infinite loop while in Pimcore iFrame

### ISSUE THREAD

**Issue by ThomasKeil** [pimcore/pimcore#1763]
# Behaviour
Displaying a broken (e.g. an error: _The template ":Default:default.html.php" does not exist_.) page renders Pimcore completely unusable since the Symfony Errorpage due to a bug executes an infinite loop.

# Steps to reproduce
Take a fresh install. Open the "Home" page. Set the action to something not existing, like "adefault". Reload the Page.

This error won't trigger when not being in pimcore.

# Code example 
```js
// vendor/symfony/symfony/src/Symfony/Bundle/TwigBundle/Resources/views/base_js.html.twig

// ...
  createToggles: function() {
// ...
    for (var i = 0; i < toggles.length; i++) {
// ... 
      for (var i = 0; i < toggleLinks.length; i++) {
//  ...
// The inner for obviously resets the outer one
```
The error is obviously in Symfony, but since it only triggers within pimcore I'm reporting it here ;-)

Please advice on how to proceed.

**Comment by maff** [comment]
This is already addressed in https://github.com/symfony/symfony/pull/23577 and should be released with the next 3.3 patch version.

In the meantime, you could manually patch your installation by overriding the template on your installation by creating a `app/Resources/TwigBundle/views/base_js.html.twig` template and adding the content from the patched file in the PR. For more details see https://symfony.com/doc/current/templating/overriding.html.

**Comment by maff** [comment]
Fixed in Symfony 3.3.6: http://symfony.com/blog/symfony-3-3-6-released

**Comment by ThomasKeil** [comment]
Works - thanks for pointing out the update.
